website/templates/project/project.mako

```diff
@@ -403,7 +403,7 @@
                 <div id="containment">
                     <div mod-meta='{
                         "tpl": "util/render_nodes.mako",
-                        "uri": "${node["api_url"]}get_children/",
```

Comment:
Was `/get_children/` being hit anywhere else? A quick grep seems to indicate that it's not.

If not, consider removing that route, method, and any tests for it. Not a blocker